src/python/WMCore/WMRuntime/Scripts/Timestamps.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python
+"""
+_Timestamps_ module designed to insert proper WM timing into WMCore FJR.
+Usage: python3 Timestamps.py --reportFile=$outputFile --wmJobTime=$wmJobTime
+where outputFile represents output FJR file, and wmJobTime represents
+elapsed time of WM job.
+
+"""
+
+import getopt
+import logging
+import os
+import pickle
+import sys
+
+
+# script options
+options = {"reportFile=": "", "wmJobStart=": 0, "wmJobEnd": 0}
+
+
+def addTimestampMetrics(config, wmJobStart, wmJobEnd):
+    """
+    Adjust timestamp metrics of provied FJR data.
+
+    :param config: input FJR data
+    :param wmJobStart: start time of WM job in seconds
+    :param wmJobEnd: end time of WM job in seconds
+    """
+    wmTiming = config.section_('WMTiming')
+    wmTiming.WMJobStart = wmJobStart
+    wmTiming.WMJobEnd = wmJobEnd
+    wmTiming.WMTotalWallClockTime = wmJobEnd-wmJobStart
+
+
+def main():
+    """
+    Main function of the script performs business logic:
+    - parse input arguments
+    - read provided input FJR
+    - adjust timestamp metrics of FJR data
+    - write out FJR
+    """
+    try:
+        opts, _ = getopt.getopt(sys.argv[1:], "", list(options.keys()))
+    except getopt.GetoptError as ex:
+        msg = "Error processing commandline args:\n"
+        msg += str(ex)
+        logging.error(msg)
+        sys.exit(1)
+
+    for opt, arg in opts:
```

Comment:
Did you miss a `opts.items()` here?

Reply:
`opts` is not a dictionary, here from python help:
```
import getopt
help(getopt)
Help on function getopt in module getopt:

getopt(args, shortopts, longopts=[])
    getopt(args, options[, long_options]) -> opts, args
...
The return value consists of two elements: the first is a list of
    (option, value) pairs;...
```